prime-router/src/main/kotlin/azure/ApiResponse.kt

```diff
@@ -0,0 +1,27 @@
+package gov.cdc.prime.router.azure
```

Comment:
Can you add some doc for the classes in this file?

Reply:
Will do